third_party/types/common/callback.go

```diff
@@ -7,7 +7,9 @@ type MotorCallback interface {
 	OnMotorEvent(msg string, isDone bool)
 }
 
-type defaultCallback struct{}
+type defaultCallback struct {
+	MotorCallback
```

Comment:
Remove this line as it silences errors whenever `defaultCallback` does not implement `MotorCallback`.